### PR TITLE
Usage standardization

### PR DESCRIPTION
**Pull request by jimmynotjim**
Fixing usage docs that weren't updated or got out of sync with the others.
## Changes
- Standardized cf-buttons usage doc
- Standardized cf-core usage doc
- Standardized cf-forms usage doc
- Standardized cf-pagination usage doc
- Standardized cf-tables usage doc
- Standardized cf-typography usage doc
## Testing
- Check out the usage files in a browser and test all the ToC links and double check I caught all the sentence case issues
## Review
- @KimberlyMunoz 
- @Scotchester 
## Checklist
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
- [x] Passes all existing automated tests
- [x] New functions include new tests
- [x] New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged
- [x] Visually tested in supported browsers and devices
- [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/cf-forms/usage.md [hunk 1]
```diff
@@ -12,19 +12,21 @@ The cf-forms component provides extensions to the basic form styles for Capital
 
 - [Variables](#variables)
     - [Color variables](#color-variables)
-    - [Sizing variables](#sizing-variables)
 - [Labels](#labels)
-    - [Label header](#label-header)
-- [Basic Form Elements](#basic-form-elements)
+    - [Basic label](#basic-label)
+    - [Label header](#label-heading)
```
Comment (Scotchester) [inline src/cf-forms/usage.md]: Should say "Label heading"

File: src/cf-forms/usage.md [hunk 2]
```diff
@@ -12,19 +12,21 @@ The cf-forms component provides extensions to the basic form styles for Capital
...
 - [Inputs](#inputs)
+    - [Input states](#basic-text-inputss)
```
Comment (Scotchester) [inline src/cf-forms/usage.md]: Should be `[Basic text inputs](#basic-text-inputs)` (note the single `s` at the end of the link, too)

File: src/cf-forms/usage.md [hunk 3]
```diff
@@ -12,19 +12,21 @@ The cf-forms component provides extensions to the basic form styles for Capital
...
-- [Groups](#groups)
-    - [Form group block](#form-group-block)
-    - [Real world example](#real-world-example)
 - [Buttons](#buttons)
-    - [Default input and button](#default-input-and-button)
+    - [Input and button](#input-and-button)
     - [Button inside input](#button-inside-input)
+- [Select dropdown](#select-dropdown)
+    - [Required select](#required-select)
+    - [Optional select](#optional-select)
```
Comment (Scotchester) [inline src/cf-forms/usage.md]: No entry for "Disabled select" here?

File: src/cf-forms/usage.md [hunk 4]
```diff
@@ -75,7 +76,6 @@ Theme variables for setting the color and sizes throughout the project. Overwrit
 </label>
 ```
 
-
```
Comment (Scotchester) [inline src/cf-forms/usage.md]: Trivial: I think two blank lines above an `##` is our unwritten standard. I would like it to be, anyway ;)

File: src/cf-tables/usage.md [hunk 5]
```diff
@@ -7,16 +7,22 @@ The [`cf-core`](../core) component is a dependency of this component."
...
+  - [Sorting type](#sorting-type)
+  - [Sorting table on page load](#sorting-table-on-page-load)
+  - [Sample sortable table](#sample-sortable-table)
+  - [Important notes about sortable tables](#important-notes-about-sortable-tables)
 - [Responsive tables](#responsive-tables)
+  - [Responsive stacked table](#responsive-stacked-table)
+  - [Responsive stacked table with header](responsive-stacked-table-with-header)
+  - [Responsive table - horizontal scroll variation](#responsive-table---horizontal-scroll-variation)
 
 ## Variables
```
Comment (Scotchester) [inline src/cf-tables/usage.md]: Should we add a `### Color variables` section in here? I think one of the files above had a subsection, even when there was only one of the subsections.

File: src/cf-typography/usage.md [hunk 6]
```diff
@@ -7,30 +7,25 @@ and has more basic typography patterns.
 ## Contents
 
 - [Variables](#variables)
-- [Pull quote](#pull-quote)
-- [Micro copy](#micro-copy)
-- [Short description](#short-description)
-- [Date](#date)
+  - [Color variables](#color-variables)
+- [Running text](#running-text)
```
Comment (Scotchester) [inline src/cf-typography/usage.md]: Is the "Running text" sectioning really useful here? If so, is it used correctly? The date atom doesn't really go in running text, and links with icons do go in running text, to call out two obvious inconsistencies.

I think I'd suggest just ditching "Running text" as a grouping.

File: src/cf-typography/usage.md [hunk 7]
```diff
@@ -7,30 +7,25 @@ and has more basic typography patterns.
...
+  - [Micro copy](#micro-copy)
+  - [Date](#date)
+  - [Pull quore](#pull-quote)
```
Comment (Scotchester) [inline src/cf-typography/usage.md]: *quote

File: src/cf-tables/usage.md (hunk 5 above)
Comment (jimmynotjim) [inline src/cf-tables/usage.md]: Updated to match cf-forms

File: src/cf-typography/usage.md (hunk 6 above)
Comment (jimmynotjim) [inline src/cf-typography/usage.md]: What should the group be? These are kind of on their own since they aren't headings, headers, or links.

Comment (Scotchester) [inline src/cf-typography/usage.md]: /shrug. "Miscellaneous"? Or just promote the subitems to top-level items.

**Comment by Scotchester** [comment]
:+1:

File: src/cf-typography/usage.md (hunk 6 above)
Comment (jimmynotjim) [inline src/cf-typography/usage.md]: Done

**Pull request merged**